frontend_app_file/test/fixture/content/revisionList.js

```diff
@@ -0,0 +1,106 @@
+export const revisionList = [{
```

Comment:
I think this could be in frontend_lib so that this fixture could be usable for any apps